ReSharper.FSharp/src/FSharp.Psi.Services/src/Util/FSharpNamingService.fs

```diff
@@ -448,7 +448,7 @@ type FSharpNamingService(language: FSharpLanguage) =
         // todo: partially applied functions?
         | :? IPrefixAppExpr as appExpr ->
             let invokedFunctionReference = appExpr.InvokedFunctionReference
-            if isNotNull invokedFunctionReference then
+            if isNotNull invokedFunctionReference && invokedFunctionReference.GetFcsSymbol() :? FSharpMemberOrFunctionOrValue then
```

Comment:
What symbols are returned for exceptions and delegates? Can there be any other cases?

Reply:
Ah, then we don't currently return other symbols from appExpr.InvokedFunctionReference.